src/stores/user.js

```diff
@@ -29,6 +32,7 @@ export const useUserStore = defineStore({
     set(user){
       this.setUserInfo(user.uid);
       this.user.auth = user;
+      this.user.docs = user;
```

Comment:
No. Do not set the user.docs to the same as auth user. It is different to the user document snap shot that is being set from setUserInfo. This just overwrites it which makes no sense.